app/components/invite-button.ts

```diff
@@ -0,0 +1,23 @@
+import type AuthenticatorService from 'codecrafters-frontend/services/authenticator';
+import Component from '@glimmer/component';
+import { inject as service } from '@ember/service';
+
+export interface Signature {
+  Element: HTMLDivElement;
+
+  Args: {
+    isCollapsed: boolean;
+    onClick: () => void;
```

Comment:
This doesn't need to be accepted as an arg, you can directly use `{{on "click" <whatever>}}` from the caller